conf/quota.yaml.template

```diff
@@ -30,3 +30,12 @@
      diskFileCount: 1000
      hdfsBytesWritten: 3G
      hdfsFileCount: 200
+
+# Users without configuration will use this configuration
```

Comment:
IMO, we can add a new field `level` such as TENANT、USER（default for compatible）instead hack the name as default, and For cluster default, we can use SYSTEM level. 

>  -  level: System
   quota:
     diskBytesWritten: 100m
     diskFileCount: 200
     hdfsBytesWritten: 200m
     hdfsFileCount: 100

Reply:
Good idea.

Reply:
cc @RexXiong   PTAL.

Reply:
I modified the hierarchy of the configuration file to make it more intuitive.